app/views/stream/_main.erb

```diff
@@ -33,6 +33,9 @@
             <button type="submit" class="btn btn-link discrete">mark as read</button>
           </form>
           <% end %>
+          <% if exercise.request_help %>
+            <button class="btn btn-xs">requested help</button>
```

Comment:
Using a button implies that it's interactive: that clicking it does something.

I was under the impression that we wanted to display the image provided in the original issue https://github.com/exercism/discussions/issues/47

![Requested help](https://cloud.githubusercontent.com/assets/6845482/16541708/addee92c-4058-11e6-9f5c-e05ab0d59e6f.png)

If I'm wrong about that, then maybe the [Bootstrap "label" component](http://getbootstrap.com/components/#labels)?